### PR TITLE
Register alecks.is-a.dev

### PR DESCRIPTION
**Pull request by Alecks20**
Register alecks.is-a.dev with CNAME record pointing to euphoria.alecks.dev.

### CODE REVIEW

File: domains/alecks.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Alecks20",
+           "email": "109296824+Alecks20@users.noreply.github.com",
```
Comment (CuteDog5695) [inline domains/alecks.json:4]: Either remove your no-reply email or replace it with your own.

**Review by iostpa — CHANGES_REQUESTED** [review]
Tried visiting the page but I can't since the dev domain doesn't have SSL so I had to check ReviewMate which seems to give page 404 not found. Can you give a preview of your website?

**Comment by Alecks20** [comment]
The website currently returns a 404 its an nginx reverse proxy, once the domain is initialized I will setup a lets encrypt cert. 

The website I'm gonna deploy onto it is also accessible at https://alecks.dev

File: domains/alecks.json (hunk 1 above)
Comment (Alecks20) [reply domains/alecks.json]: Completed this

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**